src/io/flutter/analytics/Analytics.java

```diff
@@ -112,6 +112,10 @@ public void sendTiming(String category, String variable, long timeMillis) {
     sendPayload("timing", args, null);
   }
 
+  public void sendCaughtException(String text) {
```

Comment:
`sendCaughtException()` ==> `sendExpectedException`?

We also might want to accept the location and exception as method params so we can combine the uniformly. Something like `(String location, Exception ex)`, and, `sendEvent("expected-exception", location + ":" + ex.getClass().getName());`.